### DISCUSSION TITLE
Podman Desktop - Install into alternate location

### DISCUSSION THREAD

**Discussion by whunsicker** [podman-desktop/podman-desktop#5068] [General]
Hi Team,

I am looking for the proper installer parameter to change the default location for the installation files.  Right now they are getting put in the users AppData folder, and our company requires them to be placed in the Program Files folder.  Please let me know how to best go about that using the installer exe.

Thanks

**Comment by benoitf** [comment]
hello @whunsicker it's for podman desktop itself or podman installation ?

you should add `/allusers` argument to the installer I think it should do a perMachine installation and maybe then use Program Files

**Reply by whunsicker** [reply]
AllUsers still installed the package in the users AppData folder, but changed where it placed the shortcut icon to the all users folder.

I also tried 
podman-desktop-airgap-1.4.0-setup-x64.exe /allusers /targetdir="C:\Program Files (x86)\Podman Desktop\"

but without success

Thinking I might just have the wrong parameter or something.
